bin/templates/scripts/cordova/lib/plugman/pluginHandlers.js

```diff
@@ -91,18 +93,29 @@ var handlers = {
             }
         },
         uninstall:function(obj, plugin, project, options) {
+            var podsJSON = require(path.join(project.projectDir, 'pods.json'));
             var src = obj.src;
 
             if (!obj.custom) {
                 var keepFrameworks = keep_these_frameworks;
-
                 if (keepFrameworks.indexOf(src) < 0) {
-                    project.frameworks[src] -= (project.frameworks[src] || 1) - 1;
-                    if (project.frameworks[src] < 1) {
-                        // Only remove non-custom framework from xcode project
-                        // if there is no references remains
-                        project.xcode.removeFramework(src);
-                        delete project.frameworks[src];
+                    if (obj.type === 'podspec') {
+                        if(podsJSON[src]) {
+                            if((podsJSON[src].count - 1) > 0) {
+                                podsJSON[src].count = podsJSON[src].count - 1;
+                            } else {
+                                delete podsJSON[src];
+                            }
+                        }
+                    } else {
+                        project.frameworks[src] -= (project.frameworks[src] || 1) - 1;
```

Comment:
Changed to 

`project.frameworks[src] = project.frameworks[src] || 1;
                        project.frameworks[src]--;`

for now, should definitely be refactored in the future.